src/provider.ts

```diff
@@ -18,44 +29,54 @@ export default class TaskTreeDataProvider implements TreeDataProvider<TreeItem>
     const children: TreeItem[] = [];
 
     if (workspace.workspaceFolders) {
-      var filePath = `${workspace.workspaceFolders[0].uri.fsPath}/Makefile`;
-      var commands = await extractCommands(filePath);
+      const filePath = `${workspace.workspaceFolders[0].uri.fsPath}/Makefile`;
+      let commands = await extractCommands(filePath);
+
+      if (this.config.sortAlphabetically) {
+        commands = commands.sort((a, b) => a.command.localeCompare(b.command));
+      }
 
       if (commands.length !== 0) {
-        for (let y = 0; y < commands.length; y++) {
-          children.push(new MakefileCommand(commands[y]));
+        for (const cmd of commands) {
+          children.push(new MakefileCommand(cmd.command, this.config.displayDescriptionCommentsInPanel ? cmd.comment : ''));
```

Comment:
Pass the command, and the description comment if enabled if display is enabled in the options.